### ISSUE TITLE
Using TikZ for pictures

### ISSUE THREAD

**Issue by ghost** [edmundsj/learnElectricalEngineering#14]
Not everybody has Mathematica, so I was wondering if maybe we could draw our new pictures in TikZ or something similar. TikZ happens to have a lot of what we need already defined: [circuitikz](https://github.com/circuitikz/circuitikz) has all the standard circuit symbols for example.
I'm willing to write up an installation guide for LaTeX/TikZ if that would help.

For the record, I think it's fine that we leave the Mathematica figures that have already been drawn up. But this would be more for people who can't install Mathematica for whatever reason (e.g., Linux users who aren't on Ubuntu).

**Comment by edmundsj** [comment]
Agreed, this is a good idea moving forward, Mathematica is great but also closed-source and expensive and not everyone has access to it. Can you link any favorite tutorials on using TikZ? I don't have a better alternative to TikZ, so let's roll with that. Let's save the scripts on the repository and generate all the .svg files. Once we migrate to reST/Sphinx, we can auto-generate the figures on each triggered build to make sure they stay current. Looks like someone has already written an [extension for sphinx](https://github.com/sphinx-contrib/tikz).

**Comment by ghost** [comment]
This is the best one I've found: [VisualTikZ](http://ctan.math.washington.edu/tex-archive/info/visualtikz/VisualTikZ.pdf). This is the "middle ground" tutorial. Actually, it's more of a reference, but at least it's relatively easy to find what you need.
The [TikZ/PGF manual](https://pgf-tikz.github.io/pgf/pgfmanual.pdf) itself is way too involved for what we'll need, and the [YouTube tutorials](https://www.youtube.com/watch?v=pcIzeN46ETc) don't go deeply enough. Each TikZ extension has its own manual, too. The one for circuitikz is on [this page](https://circuitikz.github.io/circuitikz/).
